app/controllers/katello/api/v2/api_controller.rb

```diff
@@ -76,18 +76,27 @@ def scoped_search(query, default_sort_by, default_sort_order, options = {})
       else
         query = query.paginate(paginate_options)
       end
+      page = params[:page] || 1
+      per_page = params[:per_page] || ::Setting::General.entries_per_page
 
+      scoped_search_results(query, sub_total, total, page, per_page)
+    rescue Exception => error
```

Comment:
[Rescue StandardError not Exception](https://robots.thoughtbot.com/rescue-standarderror-not-exception).